bluemira/geometry/parameterisations.py

```diff
@@ -986,6 +986,191 @@ def _make_control_points(p0, p3, theta0, theta3, l_start, l_end):
         return p1, p2
 
 
+class PictureFrameTools:
+    """
+    Curved Pictureframe parameterisation with straight (flat) inner limb
+
+    Parameters
+    ----------
+    var_dict: Optional[dict]
+        Dictionary with which to update the default values of the parameterisation.
+
+    """
+
+    @staticmethod
+    def _make_domed_leg(
+        axis,
+        x_out,
+        x_curve_start,
+        x_mid,
+        z_top,
+        z_mid,
+        flip=False,
+        r_c=0,
+    ):
+        """
+        Makes smooth dome for CP coils
+
+        Returns
+        -------
+        shape: BluemiraWire
+            CAD Wire of the geometry
```

Comment:
Can we be more specific?

Reply:
The description describes the shapes being parameterised, adding detail to the 'returns' section would be repeating this info no?

Reply:
I suppose I am happy with that.